### PR TITLE
Quebec Family Allowance Tax Credit

### LINKED ISSUE #301: Quebec Family allowance tax credit
Reference [https://www.rrq.gouv.qc.ca/en/programmes/soutien_enfants/Pages/soutien_enfants.aspx](url)

### PR DESCRIPTION
**Pull request by Shinuing**
Fixes #301

Abandoned the table in [Retraite Quebec](https://www.rrq.gouv.qc.ca/en/programmes/soutien_enfants/paiement/Pages/montant-selon-revenu-familial.aspx) and re-calculate by using [Taxation Act](https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langcont=en#se:1029_8_61_18) structure with values in [Retraite Quebec](https://www.rrq.gouv.qc.ca/en/enfants/naissance/paiement_soutien_enfants/Pages/montant.aspx) for 2023

My own [notes](https://docs.google.com/document/d/1IyQijGMiBVOA9sZGP2orNrezIHonGgwaHccmUHWJlPI/edit?usp=sharing) for comprehension

### CODE REVIEW

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/base_amount.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Quebec provides this base amount in its family allowance tax credit for each handicapped child.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/base_amount.yaml:1]:
```suggestion
description: Quebec provides this base amount under the family allowance tax credit for each handicapped child.
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/base_amount.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+values:
+  2023-01-01: 2_616
+metadata:
+  unit: currency-CAD
+  label: Quebec family allowance tax credit handicapped child base amount
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/base_amount.yaml:6]:
```suggestion
  label: Quebec family allowance tax credit disabled child base amount 
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/exceptional_care_tier1_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/exceptional_care_tier1_amount.yaml:1]: make and `exemptional_care_amount` folder withe tier_1 and tier_2

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/base_amount.yaml (hunk 2 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/handicapped_child_supplement/base_amount.yaml:6]: all yaml

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    threshold:
+      2023-01-01: 16
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-CAD
+  label: Quebec yearly amount for the purchase of school supplies
+  reference:
+    - title: Retraite Québec - Amounts and payment of Family Allowance
+      href: https://www.rrq.gouv.qc.ca/en/programmes/soutien_enfants/paiement/Pages/montant.aspx
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml:22]: I am not sure if I see the age breakdown in the source

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      2023-01-01: 4
+  - amount:
+      2023-01-01: 0
+    threshold:
+      2023-01-01: 16
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-CAD
+  label: Quebec yearly amount for the purchase of school supplies
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml:19]:
```suggestion
  label: Quebec family allowance school supplies amount
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/single_parent_family/five_children_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/single_parent_family/five_children_amount.yaml:1]: make an `amount` folder and in there put the `single_parent_family` and `two_parent_family` folders in it.

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier2.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_exceptional_care_tier2(Variable):
+    value_type = bool
+    entity = Person
+    label = "Quebec family allowance handicapped children requiring exceptional care Tier2"
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier2.py:7]: add a reference

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier1_child.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_exceptional_care_tier1(Variable):
+    value_type = bool
+    entity = Person
+    label = "Quebec family allowance handicapped children requiring exceptional care Tier1"
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier1_child.py:7]: add a reference

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_single_parent.py [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+            )
+        )
+
+        # number of shared custody children in the household
+        shared_custody_children = children - full_custody_children
+        shared_custody_payment = (
+            p.shared_custody_reduction
+            * no_spouse
+            * (
+                select(
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_single_parent.py:51]: separate the select statement out

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_single_parent.py [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+                    p.single_parent_family.three_children_amount.calc(income),
+                    p.single_parent_family.four_children_amount.calc(income),
+                    p.single_parent_family.five_children_amount.calc(income),
+                ],
+                default=0,
+            )
+        )
+
+        # number of shared custody children in the household
+        shared_custody_children = children - full_custody_children
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_single_parent.py:46]:
```suggestion
        shared_custody_children = max_(children - full_custody_children, 0)
```

File: policyengine_canada/tests/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_single_parent.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_single_parent.yaml:1]: make more tests for edge cases (0 children, 7 children etc.) test the formula at different points

File: policyengine_canada/tests/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.yaml:1]: create more tests for edge cases

File: policyengine_canada/tests/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_two_parent.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment_two_parent.yaml:1]: create more tests for edge cases

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.py [hunk 9]
```diff
@@ -0,0 +1,52 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_supplement(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance supplements"
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.py:7]:
```suggestion
    label = "Quebec family allowance supplement"
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.py [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        person = household.members
+
+        # check if the child is full custody of the head
+        full_custody = person("full_custody", period)
+        reduction_rate = max_(full_custody, p.shared_custody_reduction)
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.py:18]: use a where statement if there are different outcomes that are possible

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml (hunk 3 above)
Comment (Shinuing) [reply policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml:22]: Thanks for pointing that out! Added another reference https://www.rrq.gouv.qc.ca/en/enfants/Pages/supplement-achat-fournitures-scolaires.aspx

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/school_supplies_amount.yaml:1]: need another parameter for supplement age (4-17)

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/child_amount/maximum.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/child_amount/maximum.yaml:1]: rename to max.yaml and min.yaml

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/reduction_threshold/single_parent_family.yaml [hunk 11]
```diff
@@ -0,0 +1,21 @@
+description: Quebec reduces its family allowance tax credit at this rate for single parent family based on family income.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/reduction_threshold/single_parent_family.yaml:1]:
```suggestion
description: Quebec reduces its family allowance tax credit at this rate for a single parent family based on family income.
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/reduction_threshold/two_parent_family.yaml [hunk 12]
```diff
@@ -0,0 +1,21 @@
+description: Quebec reduces its family allowance tax credit at this rate for two parent family based on family income.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/reduction_threshold/two_parent_family.yaml:1]:
```suggestion
description: Quebec reduces its family allowance tax credit at this rate for a two parent family based on family income.
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/single_parent_amount/maximum.yaml [hunk 13]
```diff
@@ -0,0 +1,12 @@
+description: Quebec provides this maximum amount under the family allowance tax credit for individual has no cohabiting spouse.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/single_parent_amount/maximum.yaml:1]:
```suggestion
description: Quebec provides this maximum amount under the family allowance tax credit for individuals without a cohabiting spouse.
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier1_child.py [hunk 14]
```diff
@@ -0,0 +1,10 @@
...
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier1_child.py:7]:
```suggestion
    label = "Quebec family allowance handicapped children requiring Tier 1 exceptional care"
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier2.py [hunk 15]
```diff
@@ -0,0 +1,10 @@
...
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier2.py:7]:
```suggestion
    label = "Quebec family allowance handicapped children requiring Tier 2 exceptional care"
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py [hunk 16]
```diff
@@ -0,0 +1,55 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_payment(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance payment amount "
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py:7]:
```suggestion
    label = "Quebec family allowance payment amount"
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.py
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_supplement.py:1]: could a reference here too, to see the calculation steps

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py:1]: could you add a reference

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/single_parent_amount/minimum.yaml [hunk 17]
```diff
@@ -0,0 +1,12 @@
+description: Quebec provides this mimimum amount under the family allowance tax credit for individual has no cohabiting spouse.
+values:
+  2023-01-01: 389
+metadata:
+  unit: currency-CAD
+  label: Quebec family allowance tax credit minimum amount for single parent family
```
[the remark this answers is not in the document]
Comment (Shinuing) [reply policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/single_parent_amount/minimum.yaml:6]: Remaining unchanged, this parameter applies to single-parent families

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+    entity = Household
+    label = "Quebec family allowance payment amount"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_18"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        eligibility = household("count_children", period) > 0
```
[the remark this answers is not in the document]
Comment (Shinuing) [reply policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py]: Thanks for pointing that out!

Recheck eligible dependent child's age (under 18) and check eligible individual for this credit

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/shared_custody_multiplier.yaml [hunk 19]
```diff
@@ -0,0 +1,14 @@
+description: Quebec multiplies its family allowance tax credit by this factor for parents with shared custody of a child.
+values:
+  2023-01-01: 0.5
+metadata:
+  unit: /1
+  label: Quebec family allowance tax credit shared custody reduction rate
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/shared_custody_multiplier.yaml:6]: update

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/shared_custody_multiplier.yaml [hunk 20]
```diff
@@ -0,0 +1,14 @@
...
+  reference:
+    - title: Retraite Québec - You share custody of the child
+      href: https://www.rrq.gouv.qc.ca/en/enfants/changement_garde/Pages/obtenez_garde_partagee.aspx
+      # a child is in shared custody when the child lives alternately with each parent between 40% and 60% of the time each month.
+    - title: Retraite Québec - CalculAide for Family Allowance payments
+      href: https://www.rrq.gouv.qc.ca/en/services/services_en_ligne/soutien_aux_enfants/Pages/calcul_aide.aspx
+      # this tool assume 50% of the credit for each parent
+    # Legal reference unavaliable
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/shared_custody_multiplier.yaml:14]:
```suggestion
    # Legal reference unavailable
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/supplement/handicapped_child/exceptional_care_amount/tier1.yaml [hunk 21]
```diff
@@ -0,0 +1,12 @@
+description: Quebec provides this amount in its family allowance tax credit for each disabled child with exceptional care tier 1.
+values:
+  2023-01-01: 13_224
+metadata:
+  unit: currency-CAD
+  label: Quebec family allowance tax credit tire 1 exceptional care disabled child base amount
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/supplement/handicapped_child/exceptional_care_amount/tier1.yaml:6]:
```suggestion
  label: Quebec family allowance tax credit tier 1 exceptional care disabled child base amount 
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier2.py [hunk 22]
```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_exceptional_care_tier2(Variable):
+    value_type = bool
+    entity = Person
+    label = "Quebec family allowance handicapped children requiring Tier2 exceptional care"
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_exceptional_care_tier2.py:7]:
```suggestion
    label = "Quebec family allowance handicapped children requiring Tier 2 exceptional care"
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/supplement/handicapped_child/exceptional_care_amount/tier2.yaml [hunk 23]
```diff
@@ -0,0 +1,12 @@
+description: Quebec provides this amount in its family allowance tax credit for each disabled child with exceptional care tier 2.
+values:
+  2023-01-01: 8_796
+metadata:
+  unit: currency-CAD
+  label: Quebec family allowance tax credit tire 2 exceptional care disabled child base amount
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/supplement/handicapped_child/exceptional_care_amount/tier2.yaml:6]:
```suggestion
  label: Quebec family allowance tax credit tier 2 exceptional care disabled child base amount 
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py [hunk 24]
```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_eligibility(Variable):
+    value_type = float
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py:5]:
```suggestion
    value_type = bool
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py [hunk 25]
```diff
@@ -0,0 +1,19 @@
...
+    entity = Household
+    label = "Quebec family allowance eligible indivudual"
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py:7]:
```suggestion
    label = "Quebec family allowance eligible household"
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py [hunk 26]
```diff
@@ -0,0 +1,19 @@
...
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_8"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py:11]: This will get properly summed and cast to bool. We don't need parameters.
```suggestion
    adds = ["own_children_in_household"]
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py [hunk 27]
```diff
@@ -0,0 +1,59 @@
...
+        person = household.members
+        age_eligible = person("age", period) < p.age_eligibility
+
+        # check if each child is full custody
+        full_custody = person("full_custody", period)
+        shared_custody_multiplier = where(
+            full_custody, 1, p.shared_custody_multiplier
+        )
+
+        # method 1: calculate using max amount with reduction threshold, (C + D) − 4% (E − F) in taxation axt
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py:30]: please keep comments to 79 characters if possible
```suggestion
        # method 1: calculate using max amount with reduction threshold,
        # (C + D) − 4% (E − F) in taxation act
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py [hunk 28]
```diff
@@ -0,0 +1,59 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_payment(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance payment amount"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_18"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py:10]: Then remove from formula
```suggestion
    defined_for = "qc_fa_eligibility"
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py [hunk 29]
```diff
@@ -0,0 +1,19 @@
...
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        person = household.members
+        # is parent and resides with of the eligible dependent child
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py:16]: No age requirements or anything?

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py (hunk 29 above)
Comment (Shinuing) [reply policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py:16]: No, “eligible individual”, in respect of an eligible dependent child, at any time means an individual who at that time
(a)  resides with the eligible dependent child;
(b)  is the father or mother of the eligible dependent child;
https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_8

Age requirement applies only to dependent children, checked in `qc_fa_payment`

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/child_amount/min.yaml [hunk 30]
```diff
@@ -0,0 +1,12 @@
+description: Quebec provides this mimimum amount under the family allowance tax credit for each eligible child.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/fa/child_amount/min.yaml:1]:
```suggestion
description: Quebec provides this minimum amount under the family allowance tax credit for each eligible child.
```

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py [hunk 31]
```diff
@@ -0,0 +1,60 @@
...
+    entity = Household
+    label = "Quebec family allowance payment amount"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_18"
+    definition_period = YEAR
+    defined_for = "qc_fa_eligibility"
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        eligibility = household("qc_fa_eligibility", period)
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py:15]: don't need since it's in `defined_for`

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py (hunk 31 above)
Comment (Shinuing) [reply policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py:15]: Thank you for letting me know. I was wondering if we need to have `qc_fa_eligibility` as an input in test files?

Comment (MaxGhenis) [reply policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py]: yes you do

**Review by MaxGhenis — APPROVED** [review]
